src/experimental-bots/filler-common/orderSubscriberFiltered.ts

```diff
@@ -240,6 +240,7 @@ const main = async () => {
 	dotenv.config();
 
 	const args = parseArgs(process.argv.slice(2));
+	const driftEnv = args['drift-env'] ?? 'devnet';
```

Comment:
should this default to mainnet

Reply:
i tend to prefer devnet for safety of running it locally